packages/components/link/src/link.visualroute.js

```diff
@@ -35,5 +35,17 @@ export const component = () => (
     <Spec label="intlMessage">
       <Link to="/" intlMessage={intlMessage} />
     </Spec>
+    <ThemeProvider theme={themes.darkTheme}>
+      <Spec label="tone - inverted - with underline">
+        <Link to="/" tone="inverted">
+          An inverted label text
```

Comment:
@Rombelirk you rendered no text inside the `Link` prior. I am not sure how/what you wanted to test here in terms of font colour inversion if there is no text ;)

Reply:
🤦‍♂️